airbyte-integrations/connectors/source-bigquery/source_bigquery/spec.yaml

```diff
@@ -26,6 +29,10 @@ connectionSpecification:
       description: The contents of your Service Account Key JSON file. See the <a href=\"https://docs.airbyte.com/integrations/sources/bigquery#setup-the-bigquery-source-in-airbyte\">docs</a> for more information on how to obtain this key.
       title: Credentials JSON
       airbyte_secret: true
+    slice_range:
```

Comment:
Can we add the `examples: []` to the `slice_range`?

Reply:
thanks for the feedback @bazarnov, sure I will do that